README.md

```diff
@@ -161,6 +130,51 @@ default values.
   receive globally unique search results only once, then use this
   configuration variable, or use `setopt HIST_IGNORE_ALL_DUPS`.
 
+The following variables must be overridden before having loaded this script
```

Comment:
I'm hesitant about the "must" requirement here: it's helpful to provide these key arrays to simplify the new-user experience but currently, we don't require users to bind our widgets.  They can choose to simply load our plugin and never actually use its functionality.  This kind of basic "null" use-case should be preserved, I feel.

Reply:
Okay, so all these properties will be empty by default, in order to preserve the current "null" use-case. Let me change that and update the README accordingly.